pinc/CharacterSelector.inc

```diff
@@ -33,10 +34,11 @@ class CharacterSelector
             }
 
             foreach ($picker_set->get_subsets() as $code => $picker) {
```

Comment:
maybe a regular for loop would be more appropriate now that we are managing a counter variable.

Reply:
I think a foreach is still the right loop type because its iterating over a dictionary -- not an integer-indexed array. If PHP had `enumerate()` like python I'd say use that but it doesn't.